app/jobs/regular/notify_reviewable.rb

```diff
@@ -48,6 +48,8 @@ def count_for_group(group_id)
   end
 
   def notify(count, user_ids)
+    return if user_ids.empty?
```

Comment:
```suggestion
    return if user_ids.blank?
```
`blank?` here will work if it's `nil` too - just in case. More defensive.